frontend/src/metabase/static-viz/components/XYChart/XYChart.tsx

```diff
@@ -124,7 +131,7 @@ export const XYChart = ({
     xTicksDimensions.maxTextWidth,
     xScale.bandwidth,
   );
-  const xTicksCount = settings.x.type === "ordinal" ? Infinity : 4;
+  const xTickCount = settings.x.type === "ordinal" ? Infinity : 4;
```

Comment:
Use a similar name as already in our project e.g.
https://github.com/metabase/metabase/blob/7406d6fca01deedd71d04641b737a9861c0a03f3/frontend/src/metabase/visualizations/lib/timeseries.js#L152